doc/level1.rst

```diff
@@ -275,6 +275,23 @@ above rules, such headers will be judged invalid, and discarded by
 level 1 MUAs.  Such an update to the specification will also have
 to describe how an updated MUA will deal with multiple valid headers.
 
+A MUA MUST NOT send Autocrypt headers above 10 KiB in size
```

Comment:
In a spec, I tend to prefer to leave things in the singular, until there is a clear reason to put it in the plural.  In the line above, it could be (mis)read as suggesting that there are multiple autocrypt headers that are in aggregate bounded by 10KiB.  can we re-frame it in singular instead? (e.g. "An MUA MUST NOT send an Autocrypt or Autocrypt-Gossip header greater than 10KiB in size…")

Reply:
@dkg, leaving out the gossip part for now.